conanfile.py

```diff
@@ -1,4 +1,4 @@
-from conans import ConanFile
+from conan import ConanFile
```

Comment:
This one is fine, but there are other fixes to be done.
Got some issue with conan 2.0... not sure how I can do it for now.